source/connect/favorite-connections.txt

```diff
@@ -0,0 +1,71 @@
+.. _favorite-connections:
+
+====================
+Favorite Connections
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+|compass| allows you to save MongoDB connection configurations to
+easily reconnect to the same MongoDB deployment using the same
+specifications.
+
+Save a Favorite Connection
+--------------------------
+
+To save a favorite connection:
+
+1. Conifuring your :guilabel:`Connect To Host` form, with a
```

Comment:
Let's format these steps the way the steps in the /connect page are formatted (the circled numbers).